docs/changelog.md

```diff
@@ -5,16 +5,18 @@
 
 # Changelog
 
-## [Unreleased]
+<!-- ## [Unreleased] -->
 
-!!! note
-    All of the changes here are deployed to our current environment, even though
-    a release hasn't been made for them yet. If you want to have these updates
-    in a personal environment you'll need to install the package from git.
+<!-- !!! note -->
+<!--     All of the changes here are deployed to our current environment, even though -->
+<!--     a release hasn't been made for them yet. If you want to have these updates -->
+<!--     in a personal environment you'll need to install the package from git. -->
 
-    ```bash title="Installation command"
-    pip install git+https://github.com/European-XFEL/EXtra.git
-    ```
+<!--     ```bash title="Installation command" -->
+<!--     pip install git+https://github.com/European-XFEL/EXtra.git -->
+<!--     ``` -->
+
+## [2024.2]
```

Comment:
This needs to have a link to the tag at the bottom of the file (doesn't matter if the tag doesn't exist yet).